backend/src/entities/post.entity.ts

```diff
@@ -0,0 +1,11 @@
+import User from './user.entity';
+
+export default class Post {
+    id: string;
```

Comment:
You can set a default `id` in the constructor, making sure you never create a post without one.